data/macos.yaml

```diff
@@ -629,6 +629,36 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#Recent_Items'
 ---
+name: MacOSRemoteDesktopAdministratorSystem
+doc: Apple Remote Desktop (ARD) was first released in 2002 and is Apple’s desktop management system for software distribution, asset management, and remote assistance.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '/private/var/db/RemoteManagement/ClientCaches/*'
```

Comment:
Add `'/var/db/RemoteManagement/ClientCaches/*'`, etc. for completeness.

Reply:
Add those missing symbolic link representations.